---
cam_front:
  width: 640
  height: 480
  fps: 30
  device: /dev/auv_cam_front

  image_width: 640
  image_height: 480
  camera_name: cam_front
  camera_matrix:
    rows: 3
    cols: 3
    data: [688.903669, 0.000000, 262.505150, 0.000000, 699.623384, 210.402720, 0.000000, 0.000000, 1.000000]
  distortion_model: plumb_bob
  distortion_coefficients:
    rows: 1
    cols: 5
    data: [0.063985, -0.102582, -0.013374, -0.012679, 0.000000]
  rectification_matrix:
    rows: 3
    cols: 3
    data: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0]
  projection_matrix:
    rows: 3
    cols: 4
    data: [694.777771, 0.000000, 256.089773, 0.000000, 0.000000, 708.699646, 205.653799, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000]

cam_bottom:
  width: 640
  height: 480
  fps: 30
  device: /dev/auv_cam_bottom

  image_width: 640
  image_height: 480
  camera_name: cam_bottom
  camera_matrix:
    rows: 3
    cols: 3
    data: [998.599439, 0.0, 265.425260, 0.0, 979.419592, 240.402775, 0.0, 0.0, 1.0]
  distortion_model: plumb_bob
  distortion_coefficients:
    rows: 1
    cols: 5
    data: [0.869643, -1.060488, -0.007736, -0.062698, 0.0]
  rectification_matrix:
    rows: 3
    cols: 3
    data: [1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0]
  projection_matrix:
    rows: 3
    cols: 4
    data: [1081.186890, 0.0, 252.365375, 0.0, 0.0, 1083.032104, 238.487351, 0.0, 0.0, 0.0, 1.0, 0.0]
